src/exportable/ExportableObeysRule.ts

```diff
@@ -12,7 +12,7 @@ export class ExportableObeysRule extends fshrules.ObeysRule implements Exportabl
 
   toFSH(): string {
     return `${repeat(' ', INDENT_SIZE * this.indent)}* ${
-      this.path === '.' || this.path === '' ? '' : `${this.path} `
+      this.path === '' ? '' : `${this.path} `
```

Comment:
It's mostly to avoid an issue that can arise in situations like this one:
```
* . obeys inv-1
* . ^short = "Short"
```
With these two rules in sequence, the second rule would attempt to indent, since the previous rule's path is a match of the current rule's path. So after indenting, you'd get this:
```
* . obeys inv-1
  * ^short = "Short"
```
Which is correct. However, if we output the obeys rule with no path, we get a problem, where this:
```
* obeys inv-1
* . ^short = "Short"
```
becomes this:
```
* obeys inv-1
  * ^short = "Short"
```
Since the actual paths still match, the second rule still gets indented.

It ends up as sort of a toss-up: either keep the `.` path and enable an indent (which only matters if the user asked for indentation and the next rule can use it), or go with an empty path and have additional checks in the indentation optimizer based on what we know the path will actually output as, not what the path's value is.

Reply:
Alternatively, we could avoid having to include special logic in the indentation optimizer if the path on the Obeys rule gets changed from `.` to an empty string before any optimizers run. Then the paths won't match, and you'll get valid non-indented output.

Reply:
I think I like that last option (change Obeys rules with path "." to empty path)... Is it feasible without being messy?  Could it possibly be an optimizer itself?  It is, after all, taking valid FSH and changing it to also-valid FSH that follows best practices more closely.